quick-start/ruby.md

```diff
@@ -215,7 +216,7 @@ and most other tasks are executed on the hypertable.
     ```
 
 	<Highlight type="note">
-	The `by_range` and `by_hash` dimension builder is an addition to TimescaleDB 2.13.
+	The `by_range` and `by_hash` dimension builder is an addition to Timescale 2.13.
```

Comment:
Here, it should be `TimescaleDB` since it's referring to the actual software

Reply:
Thank you, fixed.